meta/main.yml

```diff
@@ -10,6 +10,11 @@ galaxy_info:
     versions:
     - trusty
     - precise
+    - xenial
```

Comment:
We are in the process of adding testing support for multiple OS platforms with [Molecule](http://molecule.readthedocs.io/en/stable-1.9/). Once we get to this role, we'll include tests for Xenial to ensure that things work. Until then, could you please omit `xenial` from the officially supported list since it is untested?